packages/contracts-bedrock/contracts/dispute/DisputeGameFactory.sol

```diff
@@ -61,27 +64,42 @@ contract DisputeGameFactory is OwnableUpgradeable, IDisputeGameFactory, IVersion
 
     /**
      * @inheritdoc IVersioned
+     * @custom:semver 0.0.2
      */
     function version() external pure returns (string memory) {
-        return "0.0.1";
+        return "0.0.2";
     }
 
     /**
      * @inheritdoc IDisputeGameFactory
      */
-    function gameCount() external view returns (uint256 _gameCount) {
-        _gameCount = disputeGameList.length;
+    function gameCount() external view returns (uint256 gameCount_) {
+        gameCount_ = _disputeGameList.length;
     }
 
     /**
      * @inheritdoc IDisputeGameFactory
      */
     function games(
-        GameType gameType,
-        Claim rootClaim,
-        bytes calldata extraData
-    ) external view returns (IDisputeGame _proxy) {
-        return disputeGames[getGameUUID(gameType, rootClaim, extraData)];
+        GameType _gameType,
+        Claim _rootClaim,
+        bytes calldata _extraData
+    ) external view returns (IDisputeGame proxy_, uint256 timestamp_) {
```

Comment:
My primary concern with named returns is that we're breaking convention with the rest of the codebase.
I also don't think they are always more readable. 

IMO they are most justifiable when yul is being used, and applying them in only those cases provides an opportunity to introduce them without breaking existing conventions.

Reply:
> Given that these contracts have 0 dependencies on existing contracts within contracts-bedrock and vice-versa, would you be open to allowing a few of these updated style suggestions to go through?

With that understanding, yes.